### PR TITLE
implemented toast notifications for Rider and EmployeeModal

### PR DESCRIPTION
**Pull request by aaronkang7**
### Summary 

This pull request implements toast notifications for rider and employee modals. There seems to be a problem with +Add Ride submission, but if that is fixed, the toast should work as well. The add Location form doesn't seem to have a separate modal yet so I didn't integrate the toast notification to it just yet.  

Toast notifications are automatically dismissed after 4 seconds after form submision.

This pull request is the first step towards implementing feature Foo

- [x] implemented X
- [ ] fixed Y

### Breaking Changes  

<!-- - Database schema change (anything that changes DynamoDB document structure)
<!-- - Other change that could cause problems (Detailed in notes)

### CODE REVIEW

File: frontend/src/components/EmployeeModal/EmployeeModal.tsx [hunk 1]
```diff
@@ -62,10 +67,12 @@ const EmployeeModal = () => {
       })).then(() => refreshDrivers());
     }
     closeModal();
+    setToast(true);
   };
 
   return (
     <>
+      {showingToast ? showToast('The employee has been added.') : null}
```
Comment (mg2239) [inline frontend/src/components/EmployeeModal/EmployeeModal.tsx:75]: you can just use the component itself here

File: frontend/src/components/EmployeeModal/EmployeeModal.tsx [hunk 2]
```diff
@@ -62,10 +67,12 @@ const EmployeeModal = () => {
       })).then(() => refreshDrivers());
     }
     closeModal();
+    setToast(true);
```
Comment (mg2239) [inline frontend/src/components/EmployeeModal/EmployeeModal.tsx:70]: we dont want this to show if they close out before submitting, only when a submission is successful

**Review by mg2239 — CHANGES_REQUESTED** [review]
made a few small changes cause i thought it was good to go, but noticed on the designs that the toast also shows up when the schedule is downloaded. could you add that in too?

**Comment by aaronkang7** [comment]
> made a few small changes cause i thought it was good to go, but noticed on the designs that the toast also shows up when the schedule is downloaded. could you add that in too?

**Review by mg2239 — CHANGES_REQUESTED** [review]
can you use the download icon too

**Review by mg2239 — APPROVED** [review]
oops i was looking at the wrong design, nvm looks good lol

**Comment by aaronkang7** [comment]
> oops i was looking at the wrong design, nvm looks good lol

oh, I'll just delete the commit then!

**Pull request merged**